cabal.project

```diff
@@ -29,199 +46,24 @@ constraints:
   , ListLike >= 4.7.3
   , witherable >= 0.4
 
-  -- Plutus dependency
-  , algebraic-graphs < 0.7
-
```

Comment:
Where did all these package settings/flags go? If _nowhere_, what's the logic for simply removing them? (I ran CTRL-F on the CHaP `README.md` and there was no pertinent hit.)

Reply:
Ok this must be a miss on my side.

All the `tests: false` where needed only because packages included with `source-repository-package` are considered _local_ packages (as in, the ones you are developing) and tests are automatically turned on. Now that dependencies come from a repo, they are _global_ packages and the tests are automatically off.

But now I noticed I have removed also some `flags: +asserts`, which is my bad. Let me know if it's important to reinstate them.

Reply:
> If nowhere, what's the logic for simply removing them?

One answer to this is: whoever put them there should have left a comment explaining why they were needed and under what circumstances they can be removed :) That's not there, so we're working backwards. The main thing is that as Andrea said, the `tests: False` stuff was only needed due to the `s-r-p`s, so is simply redundant now.

We should put the `+asserts` back and let the team decide if they need them.